README.md

```diff
@@ -69,6 +69,7 @@ container images and config changes are propagated to the cluster.
 |[VSHN](https://vshn.ch)|[Walmart Chile](https://www.walmartchile.cl)|[Weave Cloud](https://cloud.weave.works)|
 |[Workable](https://www.workable.com)|[Workarea](https://www.workarea.com)|[Working Group Two](https://wgtwo.com)|
 |[Yad2](https://yad2.co.il)|[Yusofleet](https://yusofleet.com)|[Zaaksysteem](https://zaaksysteem.nl)|
+|[FELFEL](https://felfel.ch/)
```

Comment:
That's great to hear - thanks @stephenshaw-felfel for letting us know!

Do you think you can sort the company names alphabetically?

Reply:
@dholbach sorry about that, I should have noticed this when I was editing the file. I have updated the commit accordingly.